packages/flutter/test/widgets/platform_view_test.dart

```diff
@@ -1149,6 +1149,44 @@ void main() {
         expect(renderObject.clipBehavior, clip);
       }
     });
+
+    testWidgets('clipBehavior test', (WidgetTester tester) async {
+      // Regressing test for https://github.com/flutter/flutter/issues/67343
+      Widget buildView(double width, double height, Clip clipBehavior) {
+        return Center(
+          child: SizedBox(
+            width: width,
+            height: height,
+            child: AndroidView(
+              viewType: 'webview',
+              layoutDirection: TextDirection.ltr,
+              clipBehavior: clipBehavior,
+            ),
+          ),
+        );
+      }
+
+      await tester.pumpWidget(buildView(200.0, 200.0, Clip.none));
```

Comment:
This test seems to rely on `RenderAndroidView` to be retained in its last size while the parent widget resizes. I can't find any description about this behavior in https://master-api.flutter.dev/flutter/widgets/AndroidView-class.html. CC @cyanglaz @amirh to double check if this is expected and whether we can rely on this.

Reply:
There is an existing test case to test this behavior:
https://github.com/flutter/flutter/blob/4cfd7d3e61022e9b01a9034fc01d8d7aa8a2a967/packages/flutter/test/widgets/platform_view_test.dart#L106-L154